### ISSUE TITLE
Support pytest 6.*

### ISSUE THREAD

**Issue by Querela** [kensho-technologies/pytest-annotate#8]
Current version of pytest is something like `6.2.3`. The package requires a pytest version lower than `6` which only allows for `pytest==5.4.3`.
Not sure whether changes are required, as the example is from 2018. This seems different:
```python
@pytest.fixture(autouse=True)
def collect_types_fixture():
    from pyannotate_runtime import collect_types
    collect_types.start()
    yield
    collect_types.stop()
```
https://github.com/dropbox/pyannotate/blob/1e7ddf09fdbc6fa37b2cf261f36a5563a0fe772a/example/example_conftest.py#L18-L23

**Comment by obi1kenobi** [comment]
That does seem like a different approach. At this point, I'm not sure if that example simply wasn't around when I wrote this plugin, or if there was a specific reason to go with another approach.

If you'd like to open a PR updating the plugin to use that approach and updating the pytest bound to `<7`, I'd be happy to approve it and publish a new version.

**Comment by Querela** [comment]
I can look into it but I'm not sure how to test it besides on some of my own projects.
And I can also not guarantee some timeline. Just updating the `setup.py` might not be enough.

**Comment by obi1kenobi** [comment]
I'm in the same boat 😄  Thanks for looking into it!

**Comment by obi1kenobi** [comment]
Closed by #10.
